Licensing: update W0019, 20, 21 (#8918)
DevExpress/devextreme-documentation: api-reference/50 Common/utils/zz Errors and Warnings/W0020.md

---
---
##### shortDescription

> Appears when the DevExtreme runtime key does not cover the version of DevExtreme used within a project.

Reviewer: What do you mean by "cover"? For example, do you mean:

`Appears when the DevExtreme runtime key does not correspond to the DevExtreme version used within a project.`

> Appears when the DevExtreme runtime key does not cover the version of DevExtreme used within a project.

Author: ```suggestion
Appears when the DevExtreme runtime key does not correspond to the DevExtreme version used within a project.
```

---
To resolve this warning, you can:

> - Use the DevExtreme version linked to your DevExpress license key (visit the DevExpress [Download Manager](https://www.devexpress.com/ClientCenter/DownloadManager/) to view a list of licensed versions).

Reviewer: ```suggestion
- Use the DevExtreme version linked to your DevExpress license key (the DevExpress [Download Manager](https://www.devexpress.com/ClientCenter/DownloadManager/) displays a list of licensed versions).
```

- [Renew your DevExpress Subscription](https://www.devexpress.com/buy/renew/). Once renewed, download and re-register your updated DevExpress license key, then run the `devextreme-license` CLI tool to regenerate the runtime key.

> For more information, refer to the following article: [Licensing](/concepts/Common/Licensing/00%20Licensing.md '/Documentation/Guide/Common/Licensing/').

Reviewer 2: let's attach a link to the sections about updating the key 
[Licensing]
https://docs.devexpress.com/GeneralInformation/405494/trial-register/set-up-your-dev-express-license-key#version-errors---registered-license-key-is-outdated

> For more information, refer to: [Registered License Key Is Outdated](https://docs.devexpress.com/GeneralInformation/405494/trial-register/set-up-your-dev-express-license-key#version-errors---registered-license-key-is-outdated)

Reviewer: ```suggestion
For additional information, refer to the following help topic: [Registered License Key Is Outdated](https://docs.devexpress.com/GeneralInformation/405494/trial-register/set-up-your-dev-express-license-key#version-errors---registered-license-key-is-outdated).
```
